leasing/enums.py

```diff
@@ -700,10 +700,29 @@ class OldDwellingsInHousingCompaniesPriceIndexType(Enum):
 
     class Labels:
         TASOTARKISTUS_20_20 = pgettext_lazy(
-            "Old dwellings in housing companies price index type",
+            "Periodic Rent Adjustment Type",
+            "Periodic Rent Adjustment 20y/20y",
+        )
+        TASOTARKISTUS_20_10 = pgettext_lazy(
+            "Periodic Rent Adjustment Type",
+            "Periodic Rent Adjustment 20y/10y",
+        )
+
+
+class OldDwellingsInHousingCompaniesPriceIndexType(Enum):
```

Comment:
I tried to make an empty class that simply inherits from the `PeriodicRentAdjustmentType`, but there came an error saying that the enumeration could not be extended for `OldDwellingsInHousingCompaniesPriceIndexType`. So I created this rather repetitive solution. I welcome a cleaner solution to the import issue.